cibuildwheel/macos.py

```diff
@@ -47,31 +47,41 @@ def call(args, env=None, cwd=None, shell=False):
 
         # if this version of python isn't installed, get it from python.org and install
         python_package_identifier = 'org.python.Python.PythonFramework-%s' % config.version
+        installation_bin_path = '/Library/Frameworks/Python.framework/Versions/{}/bin'.format(config.version)
         if python_package_identifier not in installed_system_packages:
             # download the pkg
             call(['curl', '-L', '-o', '/tmp/Python.pkg', config.url])
             # install
             call(['sudo', 'installer', '-pkg', '/tmp/Python.pkg', '-target', '/'])
 
+        # Python bin folders on Mac don't symlink python3 to python, so we do that
+        # so `python` and `pip` always point to the active configuration.
+        if os.path.exists('/tmp/cibw_bin'):
+            shutil.rmtree('/tmp/cibw_bin')
```

Comment:
I know it's just symlinking, but this feels slightly like a waste, removing and creating this folder over and over again?

Reply:
computers are fast :)

Reply:
Heh, fair enough. Not really concerned with that, but more with the fact that this might suggest for a better alternative :-) But fair enough, I can't immediately come up with one...